14.task.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+</head>
+<body>
+	<script>
+// Чи правильне це рішення задачі по умові? 
+//(тому що результат правильний, але чи відповідає він умові? ) 
+		const array = [1, 2, 3, 4, 5, 6, 7];
+		Array.prototype.remove = function removeElement(a, b) {
```

Comment:
Adding such a method to a prototype is a bad idea.

Your function accepts two arguments `a` and `b`, but in code, it uses numbers `this.splice(4, 1);`

In general, it is a good attempt to work with prototypes.

Reply:
Working example

```
const array = [1, 2, 3, 4, 5, 6, 7];

Array.prototype.remove = function remove(value) {
  const index = this.indexOf(value);

  if (index !== -1) {
     this.splice(index, 1); 
  }
};


array.remove(3);

console.log(array);
```